test/util/TestHelpers.ts

```diff
@@ -93,24 +92,22 @@ export class ResourceHelper {
     return response;
   }
 
-  public async createResource(fileLocation: string, slug: string, contentType: string, mayFail = false):
+  public async createResource(fileLocation: string, path: string, contentType: string, mayFail = false):
   Promise<MockResponse<any>> {
     const fileData = await fs.readFile(
       joinFilePath(__dirname, fileLocation),
     );
 
     const response: MockResponse<any> = await this.performRequestWithBody(
-      this.baseUrl,
-      'POST',
+      new URL(this.baseUrl.toString() + path),
```

Comment:
Let's not make a `URL` this way; can we `new URL(path, this.baseUrl)` or similar?

Reply:
i was not aware of this functionality, definitely better like this `new URL(path, this.baseUrl)`